docs/topics/simulation_setup.rst

```diff
@@ -0,0 +1,182 @@
+surfaces, this means that inner interfaces (closer to the state) must be
+fully contained by outer interfaces. In other words, for every volume in the
+ordered set of the state, interface 0, interface 1, etc.; any point in phase
+space that is inside interface :math:`i` is also inside interface :math:`i+1`.
+Importantly, this includes the state: the innermost interface must entirely
+encapsulate the state. Note that there is no way for OPS to check this for
+arbitrary state definitions; it requires user attention.
+
+Interface placement in TIS is essential to its efficiency. If interfaces are
+too far apart, the statistics in the estimate for
+:math:`P_A(\lambda_{i+1}|\lambda_i)` (see :ref:`tis-analysis`) are poor. If
+interfaces are too close together, then to span the transition, we'll need
+more interfaces. Since each interface represents an ensemble that must be
+sampled to convergence, this can badly hurt the efficiency. Typically, we
+aim for about 20-30% overlap between successive interfaces.
+
+There are approaches to `iteratively optimize the location of interfaces
+<https://doi.org/10.1063/1.3601919>`_.  These have been implemented for OPS,
+and will be added to the core code in the future.
+
+-----
+
+Getting an initial trajectory
+=============================
+
+Even with configurational sampling, initial conditions are not always
+completely trivial. A PDB entry might be missing hydrogens, and probably
+needs to be solvated (and energy-minimized). However, obtaining initial
+conditions for path sampling of rare events is much more difficult. Path
+sampling makes it possible to generate many rare trajectories once a first
+rare trajectory is given as input. But how can you get that first rare
+trajectory?
+
+The answer is, "any way that works." Some approaches include running long
+MD, or using a ratcheting approach (such as, or similar to, adaptive
+multilevel splitting) to get natural trajectories that go further and
+further toward the products.
+
+Another important idea is that the initial trajectory does not need to be a
+valid physical trajectory from the dynamics of this path ensemble. For
+example, it can come from a run at a different temperature. A non-physical
+path can be equilibrated to a physical path by performing an equilibration
+path sampling simulation before performing the production simulation.
+
+Note that a very bad initial trajectory could equilibrate to a nonsensical
+region of trajectory space, just like a bad configuration could equilibrate
+to a nonsensical metastable state in configurational Monte Carlo. Selecting
+a "nearly" physical path is important.
+
+One approach that has shown promise is to use metadynamics to obtain a first
+transition. After the first transition, metadynamics usually will not have
+significantly altered the potential energy surface near the barrier, so this
+trajectory is likely to equilibrate to a good initial trajectory.
+
```

Comment:
Another option that often works is a high temperature trajectories. However, here the dynamics is often altered dramatically, and so special attention is needed. One option is to do a quick committor analysis on the high temperature trajectories, close to the expected transition. A configuration with a non zero committor can be used as a valid good initial trajectory.

Reply:
I put this in the next section instead.